actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors to make diagnosing socket errors easier
+        logger.error "WebSocket error occurred: #{message}"
```

Comment:
Should it use the `debug` level? How likely is to this happen? Is this an exceptional case or something that would happen often?

Reply:
I guess this is an old discussion, but I'm seeing these errors some time, and I also wonder why they are logged as error rather than perhaps info or warning? it seems like something that can happen organically, e.g. the user closes the browser tab or navigates away as the connection is established?

Reply:
Do you have examples of the errors that are logged?

Reply:
We saw `WebSocket error occurred: Broken pipe` being logged.